### ISSUE TITLE
Separate and Document data.gov redirects app

### ISSUE THREAD

**Issue by jbrown-xentity** [GSA/data.gov#4272]
## User Story

In order to track and document data.gov redirects app, data.gov admins want the redirects app in it's own repository.

## Acceptance Criteria

[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]

- [ ] GIVEN changes need to be made to the redirects app \
  WHEN a team member searches for documentation and changelog history \
  THEN the information is available and straightforward \
  AND the app is contained in a repository by itself

## Background

This was created when porting over the wordpress site to a static version of the wordpress site.
https://github.com/GSA/datagov-website#www-redirects
Since we have removed datagov-website from usage (in favor of https://github.com/GSA/datagov-11ty), and we need to keep the redirect app, we want to separate out that code and logic into it's own repo so that it isn't lost to the ether.

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))

[comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
Handle GitHub Action Secrets appropriately, moving to the new repository

## Sketch

We discussed that this may have relevance to future plans of sharing styling, headers, and footers between applications. In that case, having a dedicated repository would allow us to serve static content from a shared place, and reference in different applications where appropriate. This could either be storing the new code (html/css/scss) in this shared repository, or pulling from another project (like 11ty) and building and caching locally.

However, this probably just involves porting over the following:
- github actions (deploy.yml) (will need to add necessary secrets in appropriate space)
- manifest.yml
- vars.x.yml
- redirects/nginx.conf (this can be moved to the root)

Make sure documentation of this repo is clean, and that a link to this repo is added appropriately to the data.gov wiki (it might be time for a complete refresh): https://github.com/GSA/data.gov/wiki/www.data.gov

**Comment by jbrown-xentity** [comment]
For reference, I think the coding work will take less than white noise threshold. However, the documentation cleanup will take longer...

**Comment by hkdctol** [comment]
Moving to Icebox for now.

**Comment by tdlowden** [comment]
@tdlowden will create a new repo for datagov-redirects

**Comment by robert-bryson** [comment]
Tim's new repo at https://github.com/gsa/datagov-redirects. Thanks, @tdlowden, I have access and am able to write to the repo.

**Comment by jbrown-xentity** [comment]
@btylerburton will check with cloud.gov pages team on final moving for federation.data.gov to redirects app (and eventually resources.data.gov)

**Comment by Bagesary** [comment]
Bags to follow up with Robert

**Comment by btylerburton** [comment]
We also need to create the ticket to consolidate the old `www-redirect-domains` service with the new one.

**Comment by jbrown-xentity** [comment]
New domains are hosted in datagov-redirects in cloud.gov, need to move everything over into this one out of the old redirects service into the new one (`datagov-redirects-service`). This may require changing things on 18f DNS repo, unclear. Need more research to finalize.
https://cloud.gov/docs/services/external-domain-service/

**Comment by tdlowden** [comment]
@jbrown-xentity I think this was assigned to you for awhile. What's the status and/or need here?

**Comment by jbrown-xentity** [comment]
Yeah, this fell off my radar. I think it's still worth doing long term, but isn't a priority for H2.0. I think bottom of the Queue is probably ok.
If we ever want to sharing styling and/or features between data.gov and catalog.data.gov and harvest.data.gov, we'll want to have this in place.

**Comment by tdlowden** [comment]
Ok. keeping in queue. I do think shared styling may be in the offing
